winforms/src/toga_winforms/app.py

```diff
@@ -250,7 +256,7 @@ def run_app(self):
             # in a usable form.
             self.native.ThreadException += self.winforms_thread_exception
 
-            self.loop.run_forever(self.app_context)
+            self.loop.run_forever(self, self.app_context, self.app_dispatcher)
```

Comment:
If `app_context` and `app_dispatcher` are both properties of `self`, why pass in all three as arguments? Why not just pass in `self` and use the property you need in the proactor? For that matter - if the dispatcher is a Winforms singleton - why pass it in as an argument at all?